### DISCUSSION TITLE
Why can't MATLAB extension be found?

### DISCUSSION THREAD

**Discussion by psychelzh** [posit-dev/positron#4752] [Q&A] [answered]
For vscode, I could add MATLAB extension to highlight MATLAB codes, which I cannot find in Positron. Is it possible? Or is it because of the license?

**Comment by juliasilge** [comment, marked as answer]
The Mathworks team currently only publishes [their extension](https://github.com/posit-dev/positron/wiki#installing-extensions) to the Microsoft marketplace, and [Positron uses the OpenVSX marketplace](https://github.com/posit-dev/positron/wiki#installing-extensions) for extensions. If you would like to use the MATLAB extension in Positron, we recommend that you post an issue on the extension's repo asking them to publish their extension to OpenVSX. Open VSX has a suggested [template](https://github.com/open-vsx/publish-extensions/blob/master/docs/external_contribution_request.md) to request that the authors of an extension cross-publish their extension on the Open VSX Registry.

**Reply by psychelzh** [reply]
Thank you very much. Actually, the authors of MATLAB extension do not have such a plan. Fortunately, there is a workaround put by the authors in https://github.com/mathworks/MATLAB-extension-for-vscode/issues/19.
